tests/integration/__init__.py

```diff
@@ -30,6 +30,11 @@
 except ImportError:
     pass
 
+try:
+    import salt.ext.six.moves.socketserver as socketserver
+except ImportError:
+    import socketserver
```

Comment:
Oh, look, `import salt.ext.six.moves.socketserver as socketserver` is happening in this file already, below on line 84.